core/embed/trezorhal/flash.h

```diff
@@ -82,8 +82,10 @@ extern const uint8_t FIRMWARE_SECTORS[FIRMWARE_SECTORS_COUNT];
 // note: FLASH_SR_RDERR is STM32F42xxx and STM32F43xxx specific (STM32F427)
 // (reference RM0090 section 3.7.5)
 #ifndef STM32F427xx
+#ifndef STM32F429xx
```

Comment:
Nit: `#if !defined(STM32F427xx) && !defined(STM32F429xx)` or `#if !(defined(STM32F427xx) || defined(STM32F429xx))` would be more readable IMHO.

Reply:
https://github.com/trezor/trezor-firmware/commit/ddc4be5379588242f44a9540599aee5e131100b4